iree/turbine/aot/build_actions.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import Callable
+
+from abc import abstractmethod, ABC
+from pathlib import Path
+import typing
+import types
+
+import inspect
+
+from iree.build.executor import ActionConcurrency, BuildAction, BuildContext, BuildFile
+from iree.turbine.aot.exporter import ExportOutput
+
+__all__ = [
+    "turbine_generate",
+]
+
+
+def turbine_generate(
+    generator: Callable,
+    *args,
+    name: str,
+    out_of_process: bool = True,
+    **kwargs,
+):
+    """Invokes a user-defined generator callable as an action, performing turbine
```

Comment:
consider renaming "generator callable" to "exporter callable" and/or explain that it is a callable that returns an exported torch module.

For me, "generator callable" resolves to the python language feature, "generator functions", like

```python
def fib():
  prev = 0
  cur = 1
  while True:
    yield cur
    prev, cur = cur, cur+prev
```

Reply:
sounds good & will do!